src/components/BookList/BookList.js

```diff
@@ -1,30 +1,29 @@
-import React, { useState } from 'react';
-import { v4 as uuidv4 } from 'uuid';
+import React from 'react';
+import { useSelector } from 'react-redux';
+// import booksReducer from '../../redux/books/books';
 import Book from '../Book/Book';
 import AddBook from '../AddBook/AddBook';
 import styles from './BookList.module.css';
 
 const BookList = () => {
-  function getInitialBooks() {
-    const books = [
-      { id: uuidv4(), title: 'The Hobbit', author: 'J.R.R. Tolkien' },
-      { id: uuidv4(), title: 'The Fellowship of the Ring', author: 'J.R.R. Tolkien' },
-      { id: uuidv4(), title: 'The Two Towers', author: 'J.R.R. Tolkien' },
-    ];
-    if (books) {
-      return books;
-    }
-    return [];
-  }
+  const books = useSelector((state) => state.books);
 
-  const [books] = useState(getInitialBooks());
+  // function getInitialBooks() {
+  //   const books = booksReducer(undefined, { type: 'books/GET_BOOKS' });
+  //   if (books) {
+  //     return books;
+  //   }
+  //   return [];
+  // }
+
+  // const [books] = useState(getInitialBooks());
```

Comment:
- [ ] Kindly remove all commented code from your codebase to make your codebase clean and readable 👍🏽